Nightly build failed to clone ITK

[CDash error](https://open.cdash.org/viewBuildError.php?buildid=7797404) shows failure to clone ITK with `git` protocol:
```
Cloning into 'ITK'... fatal: remote error:    The unauthenticated git protocol on port 9418 is no longer supported. Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information. Cloning into 'ITK'... fatal: remote error:    The unauthenticated git protocol on port 9418 is no longer supported. Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information. -- Had to git clone more than once:           3 times.
--
CMake Error at ITK-prefix/tmp/ITK-gitclone.cmake:66 (message):
  Failed to clone repository:
  'git://github.com/InsightSoftwareConsortium/ITK.git'


ninja: build stopped: subcommand failed.
```

Similar to [https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/issues/328](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/issues/328)

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -53,7 +53,7 @@ jobs:
       run: |
         mkdir build
         cd build
-        cmake -GNinja -DCMAKE_BUILD_TYPE=MinSizeRel ..
+        cmake -GNinja -DCMAKE_BUILD_TYPE=MinSizeRel -DSuperBuild_ITKSoftwareGuide_USE_GIT_PROTOCOL:BOOL=OFF ..
         ninja
 
     - name: Publish ITK Software Guide Book 1 artifact

